app/controllers/reviews_controller.rb

```diff
@@ -10,16 +10,16 @@ def create
 
     case @card.check_answer(review_params[:answer])
     when :correct_answer
-      redirect_to root_path, notice: 'Верно!'
+      redirect_to root_path, notice: t('flash.review_correct_answer')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.